sample

```diff
@@ -1,4 +0,0 @@
-#include<stdio.h>
```

Comment:
Why this deleted file is pushed in all pull requests?

Reply:
I created new feature branch in develop branch and I have sample file in develop branch and that sample file copied into new branch so that's why I am deleting the sample file

Reply:
If thats the case, dont delete that file, since develop had that file and you checked out a new branch from it, just do your changes, commit and push, it will not push the sample file.